web/ui/src/features/component/Table.module.css

```diff
@@ -32,3 +32,25 @@
 .table td:first-child {
   width: 2%;
 }
+
+.table tr th[data-sort-order="default"] {
+  cursor: pointer;
+}
+
+.table tr th[data-sort-order="desc"] {
+  cursor: pointer;
+  box-shadow: none !important;
+    &:before {
+      content: "▼";
+      float: right;
+    }
+}
+
+.table tr th[data-sort-order="asc"] {
+  cursor: pointer;
+  box-shadow: none !important;
+    &:before {
+      content: "▲";
+      float: right;
+    } 
+}
```

Comment:
is "box-shadow: none !important;" needed here?
can we simplify the CSS with:

.table tr th {
  cursor: pointer;
}

.table tr th[data-sort-order="desc"]:before {
  content: "▼";
  float: right;
}

.table tr th[data-sort-order="asc"]:before {
  content: "▲";
  float: right;
}

Reply:
if you remove the 'default' as stated in my other comment then I believe that this should be ok

Reply:
CSS block for `default` table header removed. I believe this can be resolved now.